src/java/picard/sam/revertsam/RevertSamUtil.java

```diff
@@ -0,0 +1,188 @@
+package picard.sam.revertsam;
+
+import htsjdk.samtools.BAMRecordCodec;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMFileWriter;
+import htsjdk.samtools.SAMFileWriterFactory;
+import htsjdk.samtools.SAMReadGroupRecord;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SAMRecordQueryNameComparator;
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.SamReaderFactory;
+import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.filter.FilteringIterator;
+import htsjdk.samtools.filter.SamRecordFilter;
+import htsjdk.samtools.util.CloserUtil;
+import htsjdk.samtools.util.FastqQualityFormat;
+import htsjdk.samtools.util.Log;
+import htsjdk.samtools.util.QualityEncodingDetector;
+import htsjdk.samtools.util.SortingCollection;
+import picard.PicardException;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Functions for creating writers, headers, and other objects needed by RevertSam.
+ */
+public class RevertSamUtil {
```

Comment:
what's the benefit of shoving these methods into a generic `Util` class that is so tied to `RevertSam` that it's in the name?